gymnasium/experimental/wrappers/rendering.py

```diff
@@ -79,9 +81,192 @@ def render(self) -> RenderFrame | list[RenderFrame] | None:
 
 
 class RecordVideoV0(gym.Wrapper):
-    """Record a video of an environment."""
+    """This wrapper records videos of rollouts.
+
+    Usually, you only want to record episodes intermittently, say every hundredth episode.
+    To do this, you can specify ``episode_trigger`` or ``step_trigger``.
+    They should be functions returning a boolean that indicates whether a recording should be started at the
+    current episode or step, respectively.
+    If neither :attr:`episode_trigger` nor ``step_trigger`` is passed, a default ``episode_trigger`` will be employed.
+    By default, the recording will be stopped once reset is called. However, you can also create recordings of fixed
+    length (possibly spanning several episodes) by passing a strictly positive value for ``video_length``.
+    """
+
+    def __init__(
+        self,
+        env: gym.Env,
+        video_folder: str,
+        episode_trigger: Callable[[int], bool] = None,
```

Comment:
`step_idx` is increased during `step`, thus `step_trigger` is checked only during `step`.
On the other hand, `episode_idx` is increased during `reset`, and `episode_trigger` is called there.
This means they cannot trigger at the same time, it can happen they trigger one after the other, in that case, the video will be just one frame.